### DISCUSSION TITLE
Revert hotkeys to latin script

### DISCUSSION THREAD

**Discussion by Tyfaspo** [ppy/osu#31212] [Q&A]
Is it possible to remove localized hotkeys, they are confusing for most users, since we don't see Cyrillic letters in hotkeys anywhere else. They look out-of-place as well. Thanks.
![image](https://github.com/user-attachments/assets/bc06f8e9-c574-4e81-b465-99d7f20170bf)

**Comment by peppy** [comment]
I think this is part of https://github.com/ppy/osu-framework/pull/5790
